src/lib.rs

```diff
@@ -3621,7 +3639,13 @@ safety_comment! {
     /// documentation once this PR lands.
     unsafe_impl!(T: ?Sized => NoCell for *const T);
     unsafe_impl!(T: ?Sized => NoCell for *mut T);
+    unsafe_impl!(T => TryFromBytes for *const T; |c: Maybe<*const T>| {
```

Comment:
Can you update the preceding safety comment to mention `TryFromBytes` in addition to `FromZeros`?

Reply:
Done!